pkg/cloudprovider/providers/gce/gce_disks.go

```diff
@@ -469,6 +476,48 @@ func (manager *gceServiceManager) getRegionFromZone(zoneInfo zoneType) (string,
 	return region, nil
 }
 
+func (manager *gceServiceManager) getProjectsAPIEndpoint() string {
```

Comment:
Why the move?

Reply:
Because this struct is defined in `gce_disks` file and it is only used in this file. I don't know why we had just these 2 functions in other file, so I moved them for being consistent.

Reply:
These methods are not unique to disks, while they are not yet used by any other consumer, they could be. So I would support leaving them where they are.

Reply:
Restored the files to their original location.